cogs/weightcog.py

```diff
@@ -56,7 +56,13 @@ class WeightCog(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.BOT = bot
         self.CONFIG: Final[Dict[str, Any]] = self.get_config()
-        self.WEIGHT_COG_DATA_PATH: Final[str] = self.CONFIG["weight-cog-data-path"]
+        self.data_folder: Final[str] = self.CONFIG["weight-cog-data-path"]
+
+    @commands.Cog.listener()
+    async def on_ready(self) -> None:
+        """Outputs the module name when the bot is ready."""
+
+        print("Module: WeightCommands")
```

Comment:
Better alternative:

print(f"Module: {self.__class__.__name__}")

Reply:
This doesn't work I am not sure why, will research on it. Leaving it untouched for now

Reply:
```
`print(f"Module: {self.__class__.__name__}")`
```